test/Datasource.test.js

```diff
@@ -57,10 +62,55 @@ describe('Test all datasouses from ../src/lib/datasouce', () => {
         });
     
         it('should resolve with null', ()=> {
-          return dataSource.delete().then((res) => {
-            expect(res).toEqual(null);  
+          return dataSource.delete('key').then((res) => {
+            expect(res).toEqual(true);  
           });
         });
       })
 
+      describe('#chaining', () => {
+        it('get shoud return null', () => {
```

Comment:
Literally, it should return Promise and resolve with a null. Same for the rest.

Also, try to keep in touch with grammar

Reply:
I changed it